### DISCUSSION TITLE
请问speech-to-text能获取到时间戳的信息吗，希望可以支持一下。

### DISCUSSION THREAD

**Discussion by qinyuanbin** [PaddlePaddle/PaddleSpeech#2718] [Q&A] [answered]
请问是否可以输出一个带时间信息的数据结构
我现在尝试了 speech asr 命令发现只返回了字符串，没有任何其他的信息了
这个是为了识别出来之后，顺便生成对应字幕文件，音字同步。否则需要人工去校对，非常的耗时。

下面是Google’s Speech-to-Text API

```js
[
    {
        "startTime": "1.400s",
        "endTime": "1.800s",
        "word": "okay"
    },
    {
        "startTime": "1.800s",
        "endTime": "2.300s",
        "word": "so"
    },
    ...
]
```

**Comment by yt605155624** [comment, marked as answer]
https://github.com/PaddlePaddle/PaddleSpeech/tree/develop/demos/streaming_asr_server

**Reply by qinyuanbin** [reply]
再问一个问题：这个流服务的时间信息，与网络延迟或者类似的因素应该不影响结果吧，比如我传输一个音频，如果有一半上传快，后面一半卡住很久才继续上传，这中间卡住的一段时间，会被记录在最终的结果中吗？

**Reply by qinyuanbin** [reply]
比如“我们”这个词语，本来间隔是1s，但是上传的时候，“我”字部分上传了，“们”字两秒后才上传，那这中间的间隔是1s还是2s？

**Comment by moosewoler** [comment]
可以先把音频切片，切完之后的能带出时间信息，然后再识别。
https://aistudio.baidu.com/aistudio/projectdetail/5114451

**Reply by qinyuanbin** [reply]
切片会影响识别的准确率吗
